__test__/users.test.ts

```diff
@@ -0,0 +1,263 @@
+import { userHandler } from "../pages/api/users";
```

Comment:
throughout this entire testing file, can we add types to each `body` or `expected` function. Lets make sure that yup and the `InferType` functions are also working properly

Reply:
do you mean replacing const body with let body: User; and then doing body = {...} ?

Reply:
Just `const body:User = {}` should work